authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -761,18 +772,36 @@ public Boolean onGroupAdded(Group group) throws AuthorizationPluginException {
 
   @Override
   public Boolean onGroupRemoved(Group group) throws AuthorizationPluginException {
-    VXGroupList list = rangerClient.searchGroup(ImmutableMap.of("name", group.name()));
-    if (list.getListSize() == 0) {
-      LOG.warn("The group({}) doesn't exist in the Ranger!", group);
+    VXGroupList vxGroupList = rangerClient.searchGroup(ImmutableMap.of("name", group.name()));
+    boolean isExist = false;
+    long groupId = -1;
+    try {
+      for (VXGroup vxGroup : vxGroupList.getList()) {
+        Class<?> clazz = vxGroup.getClass();
+        Field field = clazz.getDeclaredField("name");
+        field.setAccessible(true);
+        String value = (String) field.get(vxGroup);
+        if (group.name().equals(value)) {
+          groupId = vxGroup.getId();
+          isExist = true;
+          break;
+        }
+      }
+    } catch (Exception e) {
+      throw new AuthorizationPluginException("Fail to get the field name of class VXGroup");
+    }
```

Comment:
I think we can reuse `isGroupExist()` function. If we can return Option(long) or Long type in the `isGroupExist` function?

Reply:
OK.